src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -203,7 +203,7 @@ public Task AllowPreflightReuqest()
 
                         var nodeTag = nodeInfo.NodeTag == "?" 
                             ? null : nodeInfo.NodeTag;
-                        await ServerStore.AddNodeToClusterAsync(serverUrl, Convert.FromBase64String(nodeInfo.PublicKey), nodeTag, validateNotInTopology:false);
+                        await ServerStore.AddNodeToClusterAsync(serverUrl, Convert.FromBase64String(nodeInfo.PublicKey ?? string.Empty), nodeTag, validateNotInTopology:false);
```

Comment:
I've seen it happen when nodeInfo.NodeTag == null, then nodeInfo.PublicKey was also null. Perhaps this is not a good fix - it was quick and dirty fix because it prevented me to create cluster in studio.